docs/source/examples/cdr_qrack.md

```diff
@@ -0,0 +1,177 @@
+    return results[0]
+```
+
+## Qiskit Fake Backend
+
+CDR requires the use of a quantum device or a noisy simulator. In this example, a Qiskit 5 Qubit Fake Backend is used. The [Fake5QV1](https://docs.quantum.ibm.com/api/qiskit/qiskit.providers.fake_provider.Fake5QV1) uses configurations and noise settings taken previously from the 5 qubit IBM Quantum Yorktown device. The `qiskit_noisy` function takes the Cirq circuit and uses Mitiq to change it into a Qiskit circuit. After adding measurements, the circuit is run on the fake backend. The expectation value for `00` is then returned.
+
+```{code-cell} ipython3
+# Use Qiskit's Fave5QV1 as a noisy simulator
+def qiskit_noisy(circuit: cirq.Circuit, shots=1000):
+    """Execute the input circuit and return the expectation value of |00..0><00..0|"""
+
+    # Cirq -> Qiskit circuit
+    qiskit_circ = mitiq.interface.mitiq_qiskit.to_qiskit(circuit)
+
+    # Add measurement gates to the circuit
+    qiskit_circ.measure_all()
+
+    # Setup the fake backend and run the circuit
+    noisy_backend = Fake5QV1()
+    job = noisy_backend.run(qiskit_circ, shots=shots)
+
+    # Use the resulting counts to return the expectation value of 00
+    counts = job.result().get_counts()
+    ret_val =  counts[qiskit_circ.num_qubits * "0"] / shots
+    return ret_val
+```
+
+## Cirq Simulator for exact result
+
+The `compute_density_matrix` is the Cirq density matrix simulator with a Mitiq wrapper. It is used to obtain the exact `00` expectation value. This is then used to determine the accuracy of the mitigated and unmitigated reuslts.
+
+```{code-cell} ipython3
+def cirq_simulate(circuit: cirq.Circuit) -> np.ndarray:
+    """Returns Tr[ρ |0⟩⟨0|] where ρ is the state prepared by the circuit 
+    executed without depolarizing noise.
+    """
+    res = compute_density_matrix(circuit, noise_level=(0.0,))
+    return res[0, 0].real
+```
+
+## Executing CDR
+
+With the different executor functions defined for running the Qrack, Qiskit, and Cirq simulators, the mitiq `execute_with_cdr` function can be called.
+
+Before getting the results from using CDR, `cirq_simulate` is called to get the ideal expectation value from the circuit. Next, the circuit is run on the fake Qiskit backend, using `qiskit_noisy` in order to obtain an unmitigated expectation value for `00`. Finally, `execute_with_cdr` is called to obtain the mitigated expectation value.
+
++++
+
+```{note}
+Since we are dealing with expectation values, the `observable` parameter is set to `None` in the `execute_with_cdr` call.
+```
+
+```{code-cell} ipython3
+ideal_expval = cirq_simulate(circuit).round(5)
+print(f"Ideal expectation value from Cirq Simulator: {ideal_expval:.3f}")
+
+unmitigated_expval = qiskit_noisy(circuit)
+print(f"Unmitigated expectation value from Qiskit Fake backend: {unmitigated_expval:.3f}")
+
+mitigated_expval = cdr.execute_with_cdr(
+    circuit,
+    qiskit_noisy,
+    observable=None,
+    simulator=qrack_simulate,
+    seed=0,
```

Comment:
I think this needs to be called `random_state` in order for it to have an impact.

Reply:
@natestemen can you clarify what you mean.

Reply:
I don't think `execute_with_cdr` accepts a `seed` argument. According to the docstring it looks like it should be called `random_state`:
https://github.com/unitaryfund/mitiq/blob/357f2c0342ee68b551843f319281586422a5d1be/mitiq/cdr/cdr.py#L93

Reply:
Thanks @natestemen I've made a note that the User Guide needs to be updated with this as well.